src/components/SQFormDialog/SQFormDialogInner.js

```diff
@@ -19,12 +20,28 @@ const Transition = React.forwardRef((props, ref) => {
   return <Slide direction="down" ref={ref} {...props} />;
 });
 
+const stickyStyles = {
+  position: 'sticky',
+  background: ({palette}) => palette.background.paper,
+  zIndex: 1
```

Comment:
We generally want to avoid magic numbers for z-index's by using the z-index values available in the theme. 
Since this is inside a dialog to make an element sticky this should be harmless.